addons/mass_mailing/views/mailing_list_views.xml

```diff
@@ -108,131 +108,104 @@
-                                                    <span><field name="contact_pct_opt_out"/>%</span>
-                                                    <span class="text-muted">Opt-out</span>
-                                                </a>
-                                            </div>
-                                            <div class="col-3">
-                                                <a name="action_view_contacts_blacklisted" type="object" class="d-flex flex-column align-items-center">
-                                                    <span><field name="contact_pct_blacklisted"/>%</span>
-                                                    <span class="text-muted">Blacklist</span>
-                                                </a>
-                                            </div>
-                                        </div>
+                                    <div class="col-3 border-end">
+                                        <a name="action_view_contacts_opt_out" type="object" class="d-flex flex-column align-items-center">
+                                            <span><field name="contact_pct_opt_out"/>%</span>
+                                            <span class="text-muted">Opt-out</span>
+                                        </a>
+                                    </div>
+                                    <div class="col-3">
+                                        <a name="action_view_contacts_blacklisted" type="object" class="d-flex flex-column align-items-center">
+                                            <span><field name="contact_pct_blacklisted"/>%</span>
+                                            <span class="text-muted">Blacklist</span>
+                                        </a>
                                     </div>
                                 </div>
                             </div>
                         </div>
                         <!-- Card when the Kanban view is not grouped -->
-                        <div class="oe_kanban_global_click o_mailing_list_kanban_ungrouped d-flex gap-1 flex-row justify-content-between align-items-center flex-wrap">
+                        <div class="o_mailing_list_kanban_ungrouped d-flex gap-1 flex-row justify-content-between align-items-center flex-wrap">
                             <div class="col-lg-3 col-sm-6 col-12 py-0 my-auto">
-                                <div class="d-flex text-large">
-                                    <span class="d-inline-block text-truncate fw-normal text-large"
-                                          t-att-title="record.name.value">
-                                        <field name="name"/>
-                                    </span>
-                                </div>
+                                <span class="d-inline-block text-truncate fs-2 fw-normal lh-sm" t-att-title="record.name.value">
```

Comment:
remove the span and put the classes on the field?

Reply:
we need span for t-att-title

Reply:
actually I have tried and we can use t-att-title on a field (at least in a kanban-card)